sql/nondesktop_exact_mau28_v1.sql

```diff
@@ -0,0 +1,14 @@
+SELECT
+  submission_date,
+  CURRENT_DATETIME() AS generated_time,
+  product,
+  SUM(mau) AS mau,
+  SUM(wau) AS wau,
+  SUM(dau) AS dau
+FROM
+  nondesktop_exact_mau28_by_dimensions_v1
```

Comment:
It's not clear to me that there's a significant use case for wanting to see desktop and nondesktop together in one query, so I had assumed it would make most sense to keep these as separate tables and a motivated user could union them if needed.

Reply:
> It's not clear to me that there's a significant use case for wanting to see desktop and nondesktop together in one query

fair, I just think it would be nice organization, perhaps less clutter. that said, backfilling via write_mode=TRUNCATE would be more expensive with a merged table, and querying the two tables together is very easy.

Reply:
`_by_product` might be worthwhile. I would like to keep terminology for these final tables in line with 2019 KPIs. So "firefox_desktop", "firefox_nondesktop", and "firefox_accounts".

Perhaps these tables should be: 
- `firefox_nondesktop_exact_mau28_by_product_by_dimensions_v1`
- `firefox_nondesktop_exact_mau28_by_product_v1`

And really, these are small enough at this point that the second could be a live view on top of the first.

Reply:
okay, I didn't understand that `firefox_desktop` and `firefox_nondesktop` were matching naming with the 2019 KPIs. In light of that this seems good, and I don't care about adding a `_by_product`.